lbuild/node.py

```diff
@@ -270,7 +291,7 @@ def add_option(self, option):
         The module options only influence the build process but not the
         selection and dependencies of modules.
         """
-        if option.name in [c.name for c in self.children]:
+        if option.name in [_cw.name for _cw in self.children]:
```

Comment:
```suggestion
        if option.name in [child.name for child in self.children]:
```

Reply:
The automatic renaming something doesn't work really well for Python code